spec/features/state_file/review_page_spec.rb

```diff
@@ -128,17 +128,19 @@
   end
 
   context "AZ" do
+    before do
+      allow(Flipper).to receive(:enabled?).and_call_original
+      allow(Flipper).to receive(:enabled?).with(:show_retirement_ui).and_return(true)
+    end
```

Comment:
`Flipper.enable(:show_retirement_ui)` was working but since we ask `Flipper.enabled?` -- I think we sometimes got into a state where we got a flipper that was already stubbed before we came into this patch of code